src/utilities/index.js

```diff
@@ -18,15 +18,6 @@ export { backupValidation } from './fileSystem';
 export { debounce } from './underscoreMethods';
 export { getModalTitle, MODAL_KEYS } from './getModalTitle';
 // eslint-disable-next-line import/no-cycle
```

Comment:
this disable still needed?

Reply:
Unrelated to indicators but still needed as there is a cycle for the `finalisation` import (`UIDatabase`) 😢 .